guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -8,4 +8,30 @@
 == Upgrade Warnings
 
 [id="katello-deprecations"]
-== Deprecations
+== Removals
+
+=== Katello-Agent has been removed.
+Katello-Agent has been removed from Katello and is no longer available
+via web UI, API, or CLI. For patching hosts going forward, 
+use remote execution either in SSH or Pull mode.
+
+=== Repositories now use pulp_deb 3.0.0.
```

Comment:
I would combine this note with the one below and rename them to "Pulpcore and plugins upgraded to Pulpcore 3.28".

The note below this is actually a bugfix that happened mid-release, so it's technically not a change from 4.9 to 4.10